internal/pkg/debug/render/ssa_test.go

```diff
@@ -47,7 +47,7 @@ func TestSSA(t *testing.T) {
 		got := SSA(f)
 
 		if diff := cmp.Diff(want, got); diff != "" {
-			t.Errorf("render.SSA diff (-want +got):\n%s", diff)
+			t.Errorf("render.SSA(%s) diff (-want +got):\n%s", f.Name(), diff)
```

Comment:
While fixing the test case, it occurred to me that this error message was missing something important...